### DISCUSSION TITLE
Change item description 'on-the-fly'

### DISCUSSION THREAD

**Discussion by firstfilterservice** [opensourcepos/opensourcepos#3793] [General]
I think I might be losing my mind, but I seem to remember trying the live demo a couple months ago and I was able to edit a product description while in a sale and have that edited description save in that customer's purchase history.

What we're doing is selling oil filters for cars/trucks/etc. We get all of the filters on the counter and edit the standard description "Oil Filter Spin-on" to say "2021 Toyota Corolla Oil Filter" and then the customer knows which filter goes on which car.

If it's saved in the custom's account, next time he's in we can look back in his purchase history and we don't have to look each car up in the catalog again.

I can't seem to make the live demo do that now. Am I going crazy?!

**Comment by odiea** [comment]
In the Item dropdown did you enable the following Allow Alternate Description? Maybe this is what you are asking to do?

**Reply by firstfilterservice** [reply]
I bet that's it - the online demo is probably locked right now so I can't change that setting. I'll try to install a local copy and play around with that. I actually set up a Proxmox box (I love saying that!) last weekend to test this out and when I couldn't change the description I thought I was going nuts! Thank you!
